projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -518,6 +519,21 @@ function ( $parent_file ) use ( $users_slug ) {
 					return 'users.php' === $parent_file ? $users_slug : $parent_file;
 				}
 			);
+		} else {
+			remove_menu_page( 'profile.php' );
```

Comment:
I moved this from the `WPcom_Admin_Menu` class, since it's always needed by all sites.